netci.groovy

```diff
@@ -4,6 +4,8 @@
 import jobs.generation.Utilities;
 def project = GithubProject
 
+buildTimeLimit = 120
```

Comment:
Yes, sorry, the _second_ statement should have started with "With def". Adding def makes the thing "scoped". We want it global, so we remove the def.

Yes, if I get your thumbs up, I'll merge this on Monday morning.